src/main/java/org/launchcode/liftoffproject/controllers/AuthenticationController.java

```diff
@@ -151,4 +151,7 @@ public String logout(HttpServletRequest request){
         return "redirect:/";
     }
 
+    public ChildUser getChildUserFromSession(HttpSession session) {
+        return null;
```

Comment:
Since this only returns `null` we should either cut it entirely or refactor it. I think we could mimic the behavior for `getParentFromSession`—the question is do we want to return the User data or the Child data?